tasks/sep-dev-0/006-holocene-sys-cfg/.env

```diff
@@ -0,0 +1,4 @@
+ETH_RPC_URL="https://ethereum-sepolia.publicnode.com"
+SUPERCHAIN_CONFIG_ADDR=0xC2Be75506d5724086DEB7245bd260Cc9753911Be
```

Comment:
Also the wrong address but doesn't seem to be used in either case so can just be removed.